app/register/page.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test, expect } from '@playwright/test';
+
+const goodUser = {
+  email: 'test1@email.com',
+  password: 'test12345',
+  confirmPassword: 'test12345',
+};
+
+const badUser = {
+  email: 'wrongemail@email.com',
```

Comment:
Shashi's gonna ask you to alphabetize this I'd imagine...

Reply:
yes sir

Reply:
@vmaineng, the key values inside `badUser` still need to be sorted alphabetically.